types/chrome-remote-interface/chrome-remote-interface-tests.ts

```diff
@@ -41,6 +46,13 @@ import CDP = require('chrome-remote-interface');
                 await CDP.Close({ ...cdpPort, id: target.id });
             }
         }
+
+        assertType<Promise<void>>(client.send('Network.enable'));
+        assertType<Promise<Protocol.Page.NavigateResponse>>(client.send('Page.navigate', {url: 'https://github.com'}));
+        assertType<Promise<Protocol.Page.NavigateResponse>>(client.send('Page.navigate', {url: 'https://github.com'}, 'sessionId'));
+        client.send('Page.navigate', (error: boolean, response: Protocol.Page.NavigateResponse | Error) => {});
```

Comment:
The error type here isn't your normal JavaScript Error:

Test 1: [v8/third_party/inspector_protocol/crdtp/dispatch_test.cc#L286-L292](https://github.com/v8/v8/blob/3ff1904941fa3cbfcf932464cb146e0e3b0bb426/third_party/inspector_protocol/crdtp/dispatch_test.cc#L286-L292)
Test 2: [v8/third_party/inspector_protocol/crdtp/dispatch_test.cc#L368-L371](https://github.com/v8/v8/blob/3ff1904941fa3cbfcf932464cb146e0e3b0bb426/third_party/inspector_protocol/crdtp/dispatch_test.cc#L368-L371)
Code: [v8/third_party/inspector_protocol/crdtp/dispatch.cc#L278-L288](https://github.com/v8/v8/blob/3ff1904941fa3cbfcf932464cb146e0e3b0bb426/third_party/inspector_protocol/crdtp/dispatch.cc#L278-L288)

From the above, it appears the type should be:

```ts
interface Error {
    code: number;
    message: string;
    data?: string;
}
```

You can actually see code -32601 by doing something like the following:
```ts
client.send('Page.navigat', {url: 'https://github.com'}, (...args) => console.log(args));